src/feature_extractor/hr_to_csv.py

```diff
@@ -0,0 +1,243 @@
+from ..features.hr.video_wrapper import VideoWrapper
+from ..features.hr.ica_estimator import ICAEstimator
+import argparse
+import pandas as pd
+
+
+AMOUNT_OF_FEATURES = 7
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description="Main script for ica method testing.")
+
+    parser.add_argument(
+        "-v", "--video-path",
+        type=str,
+        required=True,
+        help="Path to the video input.")
+
+    parser.add_argument(
+        "-o", "--output-path",
+        type=str,
+        required=False,
+        help="Path to output csv file where heart rate will be appended.")
+
+    parser.add_argument(
+        "-d", "--detector",
+        type=str,
+        required=True,
+        help="Face detector method.",
+        choices=["dlib", "mtcnn", "mtcnn_kalman"])
+
+    parser.add_argument(
+        "-e", "--extractor",
+        type=str,
+        required=True,
+        help="Video extractor.",
+        choices=["opencv", "skvideo"])
+
+    parser.add_argument(
+        "-tr", "--type-roi",
+        type=str,
+        required=True,
+        help="Method for ROI extraction.",
+        choices=["skin_adapt", "skin_fix", "rect"])
+
+    parser.add_argument(
+        "-sta", "--skin-thresh-adapt",
+        type=float,
+        help="Skin threshold needed when \
+        type-roi=skin_adapt.")
+
+    parser.add_argument(
+        "-stf", "--skin-thresh-fix",
+        type=float,
+        help="Skin threshold needed when \
+        type-roi=skin_fix.")
+
+    parser.add_argument(
+        "-rc", "--rect_coords",
+        type=list,
+        help="ROI coordinates, needed when \
+            type-roi=skin_fix and no \
+            rect-regions is provided.")
+
+    parser.add_argument(
+        "-rr", "--rect-regions",
+        type=list,
+        help="ROI regions, needed when type-roi=rect \
+            and no rect-coords is provided.",
+        choices=["forehead", "lcheek", "rcheek", "nose"])
+
+    return parser.parse_args()
+
+
+def ica_estimator(args):
+    """Method that generates heart rate estimations for a given
+    input video, based in ICA rPPG method (check ica_estimator.py
+    for more details).
+
+    Args:
+        args (argparse.Namespace): object containing Heart Rate
+            estimator parameters (more details in video_wrapper.py).
+
+    Returns:
+        list: column-like array containing heart rate estimations,
+            amount of rows matches the input video duration in
+            seconds.
+
+    """
+    face_extractor = VideoWrapper(
```

Comment:
Also I believe its better if we add a space between documentation and code (and remove that extra blank line):

```suggestion
    """

    face_extractor = VideoWrapper(
```